web/explorer/src/components/NavBar.vue

```diff
@@ -0,0 +1,31 @@
+<script setup>
+import { ref } from "vue";
+
+import Menubar from "primevue/menubar";
+
+const items = ref([
+    {
+        label: "Import Analysis",
+        icon: "pi pi-file-import",
+        // TODO(s-ff): This is not the conventinal way of navigating to a new page.
```

Comment:
can you add an issue for this? is there something to change or just an info here?

Reply:
@mr-tz the conventional way of returning to the home page is by using `vue-router`'s `router.push('/')` without having to access `window`. But since we are using hash-based web history doing this does not result in reloading the page. Manually setting `window.location` is not a major issue and works fine.